src/coreclr/jit/morph.cpp

```diff
@@ -17766,11 +17769,18 @@ void Compiler::fgExpandQmarkForCastInstOf(BasicBlock* block, Statement* stmt)
     cond1Block->bbJumpDest = remainderBlock;
     cond2Block->bbJumpDest = remainderBlock;
 
-    // Set the weights; some are guesses.
+    // Currently, we don't instrument internal blocks, so the only way we can set weights to these blocks
+    // is to analyze successors and take a guess.
+    float cond2BlockWeight = 0;
```

Comment:
This is a relative probability so perhaps name it like so: (?)
```suggestion
    BasicBlock::weight_t cond2BlockLikelihood= 0;
```